tools/convert/main.cpp

```diff
@@ -90,11 +90,12 @@ static std::string GetOutputFileName(const gfxrecon::util::ArgumentParser& arg_p
     }
     else
     {
-        output_filename   = input_filename;
-        size_t suffix_pos = output_filename.find(GFXRECON_FILE_EXTENSION);
-        if (suffix_pos != std::string::npos)
+        output_filename = input_filename;
+        auto dot_pos    = output_filename.find_last_of(".");
+        auto slash_pos  = output_filename.find_last_of("\\");
```

Comment:
This is windows-specific. Slash goes the other way on Linux.

There is `kPathSep` and also path functions in `framework/util/file_path.h` like:
```
std::string InsertFilenamePostfix(const std::string& filename, const std::string& postfix);
std::string GetBasedir(const std::string& path);
std::string GetFilename(const std::string& path);
std::string GetFilenameStem(const std::string& path);
std::string GetFilenameExtension(const std::string& path);
```

Reply:
Sorry I forgot about them when I read your previous commit but this slash reminded me.

Reply:
Thank you. Fixed.